ssh/tailssh/incubator.go

```diff
@@ -438,8 +438,7 @@ func trySU(dlogf logger.Logf, ia incubatorArgs) (handled bool, err error) {
 	}
 
 	dlogf("logging in with %s %q", su, loginArgs)
-	cmd := newCommand(ia.hasTTY, su, loginArgs)
-	return true, cmd.Run()
+	return true, unix.Exec(su, loginArgs, os.Environ())
```

Comment:
hah. unix.Exec hiding over in a secondary result and relying on its result value on failure only is a hiding it a bit too much IMO

I'd pull it up a bit to make a big deal out of the exec and its err, like

```go
     dlogf("logging in with %s %q", su, loginArgs)

     // In the successful case, we never make it past this:
     err := unix.Exec(su, loginArgs, os.Environ())
    
     // If we made it this far, the exec failed.
     return true, err
```